apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js

```diff
@@ -34,9 +34,11 @@ function SiteDescriptionEdit( {
 
 	return (
 		<Fragment>
-			<PlainText
+			<RichText
 				className={ className }
 				value={ option }
+				tagName="p"
+				formattingControls={ [] }
```

Comment:
Ah interesting, does this prevent keyboard commands for applying things like bold styling?

Reply:
Looks like that's not the case. Let's stick with the PlainText component and update editor styling to better match the block selected state (with selection) vs preview state. Let us know if this is tricky to find, we'd be happy to help.

<img width="611" alt="Screen Shot 2019-08-01 at 2 54 23 PM" src="https://user-images.githubusercontent.com/1270189/62330109-8a397600-b46c-11e9-94bc-84026c7f5167.png">

The description is going to be plain text, so any html content will be escaped like:

<img width="657" alt="Screen Shot 2019-08-01 at 2 57 13 PM" src="https://user-images.githubusercontent.com/1270189/62330165-ba811480-b46c-11e9-98de-eb4270f9b219.png">

Reply:
@gawarr So `formattingControls={ [] }` is supposed to remove formatting controls from the RichText block - is that a bug where it doesn't work on wpcom? I used this specifically to get rid of the escaped content issue :(

Reply:
I'm testing on gutenberg master so maybe it's a new feature?

Reply:
> I'm testing on gutenberg master so maybe it's a new feature?

Can maybe verify by checking out back to v6.1.1 though I wouldn't assume so?